src/content/getusermedia/getdisplaymedia/js/main.js

```diff
@@ -40,7 +48,12 @@ function errorMsg(msg, error) {
 
 const startButton = document.getElementById('startButton');
 startButton.addEventListener('click', () => {
-  navigator.mediaDevices.getDisplayMedia({video: true, audio: true})
+  const options = {audio: true, video: {}};
```

Comment:
As we may be updating `options` after, we should use `let`.
```suggestion
  let options = {audio: true, video: true};
```

Reply:
JS semantics: you can change properties of a const map 🤷‍♂️

Reply:
> JS semantics: you can change properties of a const map 🤷‍♂️

I usually think of it as the difference between a const-ptr and a ptr-to-const, then it makes sense.